client/src/plugins/camunda-plugin/start-instance-tool/__tests__/StartInstanceToolSpec.js

```diff
@@ -900,8 +900,8 @@ describe('<StartInstanceTool>', () => {
 
       const actionSpy = sinon.spy(),
             actionTriggered = {
-              emitEvent: 'emit-event',
-              type: 'deployment.done',
+              emitEvent: 'open-log',
+              type: null,
```

Comment:
Why does the notification type become `null`?

Reply:
Basically the events that are defined at the very root of the application are defined as `event-name` while those which go through our event classes as `emit-event:event.type`. I've defined open-log along with the log event, to be consistent, which follows the legacy pattern (in any case the log is due for a rework some time in the future). 

The actual type we have here is 'undefined' but I've modified the mock constructor to return null to be more explicit (line 1224) I could also return undefined but I felt in this case since we expect no type specifically, null is suited.

Reply:
Then couldn't we omit the `type: null` here?

Reply:
Yeah it should work since we're using `==` in the mock. I'll just change it to return undefined instead though.

Reply:
The mock I mean, I'll implement your suggestion.